cookbooks/aws-parallelcluster-slurm/recipes/config/config_login.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: config_compute
+#
+# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: rename, find a better name that include login nodes
+setup_munge_compute_node
+
+include_recipe 'aws-parallelcluster-slurm::mount_slurm_install_dir'
```

Comment:
mount_install_dir ? --> slurm is already in the name of the cookbook.

Reply:
in the slurm cookbook we are also installing other libraries and tools (munge, pmix ... ) 

I was inclined to remove `install` and just have `mount_slurm_dir`. ..but I'd keep the `slurm` part to be clear about witch folder we are mounting.

Reply:
I agree, thanks for the clarification!